src/rsvc/srv_common.c

```diff
@@ -259,7 +260,28 @@ ds_rsvc_get_attr(struct ds_rsvc *svc, struct rdb_tx *tx, rdb_path_t *path,
 		goto out_iovs;
 
 	rc = attr_bulk_transfer(rpc, CRT_BULK_PUT, local_bulk, remote_bulk,
-				0, key_length, bulk_size - key_length);
+				0, key_length, count * sizeof(*sizes));
+	if (rc != 0)
+		goto out_iovs;
+
+	local_offset = count * sizeof(*sizes);
+	remote_offset = key_length + count * sizeof(*sizes);
+
+	for (i = 1; i < sgl.sg_nr; i++) {
+		daos_size_t size;
+
+		size = min(sgl.sg_iovs[i].iov_len,
+				       sgl.sg_iovs[i].iov_buf_len);
```

Comment:
```suggestion
			   sgl.sg_iovs[i].iov_buf_len);
```

Reply:
if you need to refresh, fix this one.   Not a huge issue but indent does look a bit off.